src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -632,7 +630,10 @@ public class C
             void verifyCompilation(CSharpCompilation compilation)
             {
                 var members = memberFunc(compilation);
-                AssertEx.Equal(expectedNullabilities, members.Select(nullabilityFunc));
```

Comment:
>AssertEx.Equal(expectedNullabilities, members.Select(nullabilityFunc)); [](start = 16, length = 71)

Why change? The loop will only report the first failure rather than all.

Reply:
It looks like there is a `FormatException` in `GetAssertMessage<T>()`. We should use `typeof(T) == typeof(byte)` there rather than `expected is IEnumerable<byte>`.

---
In reply to: [305590024](https://github.com/dotnet/roslyn/pull/37322#discussion_r305590024) [](ancestors = 305590024)